<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reportes</class>
 <widget class="QMainWindow" name="Reportes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estadísticas sobre las tortugas y las zonas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 134, 137)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableView" name="tvDensidad">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>511</width>
       <height>192</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDensidad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>721</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>A CONTINUACIÓN SE PRESENTA LA DENSIDAD POBLACIONAL
DE CADA UNA DE LAS ZONAS REGISTRADAS DENTRO DEL SISTEMA
SE PIDE PRESTAR ATENCIÓN A LA INFORMACIÓN</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbStadistics">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>711</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Estadísticas de las tortugas</string>
     </property>
     <widget class="QLabel" name="lbZone">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Zona de la tortuga</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbSensor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Sensor de la tortuga a elegir</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboZone">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>37</y>
        <width>431</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboSensor">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>67</y>
        <width>371</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneIdTortuga">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneInvestigador">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbIdTortuga">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Id Tortuga</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbNombreTortuga">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre tortuga</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneNombreTortuga">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneZona">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbZona">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Zona</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbSubespecie">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Subespecie</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneSubespecie">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneEstado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbMunicipio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Municipio</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneMunicipio">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>240</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbInvestigador">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Investigador</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbEstado">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Estado</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDistance">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Distancia:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbVelocidad">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>190</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Velocidad:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbHour">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>220</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Horas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDays">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>250</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Días:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbZones">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>260</y>
       <width>271</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#pbZones {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbZones:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>Puntos de zonas protegidas y 
no protegidas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbZones2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>390</y>
       <width>271</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#pbZones2 {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbZones2:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>Zona de la tortuga marcada</string>
     </property>
    </widget>
    <zorder>lbDensidad</zorder>
    <zorder>tvDensidad</zorder>
    <zorder>gbStadistics</zorder>
    <zorder>pbZones</zorder>
    <zorder>pbZones2</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
